tests/dialects/test_redshift.py

```diff
@@ -181,6 +181,16 @@ def test_identity(self):
         )
 
     def test_values(self):
+        self.validate_all(
+            "SELECT * FROM (VALUES (1, 2)) AS t",
```

Comment:
This would previously cause an error if we were to transpile it to redshift:

```python
    for value, column_name in zip(row, expression.args["alias"].args["columns"])
TypeError: 'NoneType' object is not iterable
```